apps/design/backend/migrations/1761063539949_add-exported-elections-table.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * @type {import('node-pg-migrate').ColumnDefinitions | undefined}
+ */
+exports.shorthands = undefined;
+
+/**
+ * @param pgm {import('node-pg-migrate').MigrationBuilder}
+ * @param run {() => void | undefined}
+ * @returns {Promise<void> | void}
+ */
+exports.up = (pgm) => {
+  // Create table to store exported election blobs for Live Reports validation
+  pgm.createTable('exported_elections', {
```

Comment:
>  We could add it to the current elections table but I opted for a new one so we can keep all exported elections, not just the most recent, although for the VxQR use case I don't need that.

Can you share more about your thinking here? I'm not immediately able to think of a use case for checking an exported election other than the most recent.